app/furniture/marketplace/management_component.html.erb

```diff
@@ -0,0 +1,67 @@
+<div class="flex flex-wrap items-baseline">
+  <h2 class="ml-2 mt-1">
+    <%= t('marketplace.marketplace.manage_header') %>
+  </h2>
+  <p class="ml-2 mt-1 truncate text-sm text-gray-500">
+    <%= marketplace.room.name %>
+  </p>
+</div>
+<%= render CardComponent.new(classes: "flex mt-3 px-0") do %>
+  <div class="flex md:w-1/5 flex-col">
+    <nav class="flex grow flex-col gap-y-5 overflow-y-auto border-r border-gray-200 p-2 md:p-5">
+      <ul role="list" class="list-none pl-0">
+
+        <%- if policy(marketplace).edit? %>
+          <%= render Marketplace::ManagementMenuItemComponent.new(location: marketplace.location(:edit)) do |menu_item| %>
+            <% menu_item.with_icon(variant: ::Icons::GearComponent) %>
+            <% menu_item.with_title(t('marketplace.marketplace.configuration_menu')) %>
+          <% end %>
+        <% end %>
+
+        <%- if policy(marketplace).edit? %>
+          <%= render Marketplace::ManagementMenuItemComponent.new(
+            location: marketplace.location(child: :products)) do |menu_item| %>
+            <% menu_item.with_icon do %>
+              <path stroke-linecap="round" stroke-linejoin="round" d="M9.568 3H5.25A2.25 2.25 0 003 5.25v4.318c0 .597.237 1.17.659 1.591l9.581 9.581c.699.699 1.78.872 2.607.33a18.095 18.095 0 005.223-5.223c.542-.827.369-1.908-.33-2.607L11.16 3.66A2.25 2.25 0 009.568 3z" />
+              <path stroke-linecap="round" stroke-linejoin="round" d="M6 6h.008v.008H6V6z" />
+            <% end%>
+            <% menu_item.with_title(t('marketplace.products.index.link_to')) %>
+          <% end %>
+        <% end %>
+
+        <%- if policy(marketplace.delivery_areas.new).create? %>
+          <%= render Marketplace::ManagementMenuItemComponent.new(
+              location: marketplace.location(child: :delivery_areas)) do |menu_item| %>
+            <% menu_item.with_icon do %>
+              <path stroke-linecap="round" stroke-linejoin="round" d="M9 6.75V15m6-6v8.25m.503 3.498l4.875-2.437c.381-.19.622-.58.622-1.006V4.82c0-.836-.88-1.38-1.628-1.006l-3.869 1.934c-.317.159-.69.159-1.006 0L9.503 3.252a1.125 1.125 0 00-1.006 0L3.622 5.689C3.24 5.88 3 6.27 3 6.695V19.18c0 .836.88 1.38 1.628 1.006l3.869-1.934c.317-.159.69-.159 1.006 0l4.994 2.497c.317.158.69.158 1.006 0z" />
+            <% end%>
+            <% menu_item.with_title(t('marketplace.delivery_areas.index.link_to')) %>
+          <% end %>
+        <% end %>
+
+        <%- if policy(marketplace.tax_rates.new).create? %>
+          <%= render Marketplace::ManagementMenuItemComponent.new(
```

Comment:
Sometimes I like to pull these into little component helpers. `Marketplace::EditComponent.menu_item(:tax_rates) do |menu_item| ...`